src/Popup.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import "./Popup.css"
```

Comment:
component names say Popup, so this component should be generic and can handle all kinds of Popup uses. So Instead of writing heading static, pass it like a prop @Kunal-inno